packages/icon/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export namespace SVGR {
```

Comment:
Rather than declaring these namespaces in a definition file like we were previously, I moved them here so we can import the namespace directly, and not include a triple-slash reference directive in `index.ts`.

Reply:
Probably better in general to do this!

Reply:
That's the hope 😁